sandbox/settings.py

```diff
@@ -95,6 +95,24 @@ class Base(DRFMixin, MagnifyCoreConfigurationMixin, Configuration):
 
     AUTH_USER_MODEL = "core.User"
 
+    AUTH_PASSWORD_VALIDATORS = [
+        {
+            "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
+        },
+        {
+            "NAME": "django.contrib.auth.password_validation.MinimumLengthValidator",
+            "OPTIONS": {
+                "min_length": 8,
```

Comment:
Add the ability to override this value with an environment variable using `values.Value`

Reply:
A value does not work for it triggers an error when django tries to compare an integer and an intergerValue. It needs an integer